src/Translation/French/translation.json

```diff
@@ -247,7 +244,7 @@
   "navigationMenu": {
     "myAccount": "Mon Compte",
     "mySendRequests": "Mes Demandes d'Envoi",
-    "signIn": "Se Connecter",
+    "signIn": "Se connecter",
     "blog": "Blog",
     "support": "Contactez-nous",
     "adminWindow": "Fenêtre d'Admin",
```

Comment:
Same here: Should we keep things lower case like in regular text? "Fenêtre d'admin", "Mes demandes d'envoi", "Mon compte", etc.?